models/task.go

```diff
@@ -211,10 +211,6 @@ func FinishMigrateTask(task *Task) error {
 	if _, err := sess.ID(task.ID).Cols("status", "end_time").Update(task); err != nil {
 		return err
 	}
-	task.Repo.Status = RepositoryReady
-	if _, err := sess.ID(task.RepoID).Cols("status").Update(task.Repo); err != nil {
```

Comment:
Why are you removing status update?

Reply:
@techknowlogick because it's the uploaders job now:
https://github.com/go-gitea/gitea/pull/12244/files#diff-9836f5da881d6313c7491648f0ef773ae33889a494e3202a485e5dbd5a45f1ebR22

Reply:
ah. Thanks :)